octopusdeploy/schema_tag_set.go

```diff
@@ -62,13 +84,44 @@ func getTagSetDataSchema() map[string]*schema.Schema {
 	}
 }
 
+func getTagSchemaForTagSet() map[string]*schema.Schema {
```

Comment:
the `getTagSchema()` from `schema_tag.go` has a few extra values that is not needed here.